### ISSUE TITLE
Feature request: evaluate multipole expansion on single axis

### ISSUE THREAD

**Issue by maroba** [maroba/multipoles#12]
### Discussed in https://github.com/maroba/multipoles/discussions/10

<div type='discussions-op-text'>

<sup>Originally posted by **nordic-node** July 14, 2023</sup>
Hi,

instead of evaluating the multipole expansion point by point, like

```python
phi = np.array([mpe(r, 0, 0, l_max=l_max) for r in rr])
``` 

where `mpe` is a multipole expansion object.

I would rather like to write something like this:

```
phi = mpe(rr, 0, 0, l_max=l_max)
```

As far as I can see, the call function just allows to pass a point. and the getitem function wants slices or masks on the same grid
that was used for the charge distribution. Is there a possibility to do the same for outside of that grid?

Cheers, Sara
</div>

**Comment by nordic-node** [comment]
Let me try this one, myself, please.

**Comment by maroba** [comment]
Sure, go ahead! If you need help, just ask. :-)

**Comment by maroba** [comment]
Closed by PR #13
